ucx.pc.in

```diff
@@ -7,5 +7,7 @@ includedir = @includedir@
 Name: @PACKAGE@
 Description: Unified Communication X Library
 Version: @VERSION@
-Cflags: -I${includedir}
+Cflags: -I${includedir} 
+Cflags.private: -I${includedir} 
```

Comment:
does `Cflags.private` really exists? What would that mean if headers are needed for build regardless of linking scenario?

Reply:
http://manpages.ubuntu.com/manpages/impish/man5/pc.5.html